lib/controller.dart

```diff
@@ -542,6 +548,29 @@ void command(UIAction action, Data data) async {
       _configRaw = configJSON;
       _config = model.Config.fromData(_configRaw);
       break;
+    case UIAction.copyToClipboardChartConfig:
+      var d = data as CopyToClipboardChartConfigData;
+      var values = _uniqueFieldCategoryValues[d.key].toList();
+      var valuesConfig = values
+          .map((value) => model.Field()
```

Comment:
why are we creating a new chartConfig? I believe we should already have one that we load when the app starts?

Reply:
This is for a utility to help the user create chartConfigs for the first time. So for each of the field categories, I'm creating a new chartConfig out of all the field values pre-filling them with defaults. I then convert the chartconfig to JSON that can directly be pasted into the codemirror. 

So in essence this is creating new chart configs purely to help speed up authoring the chartConfig JSON for the first time.

Reply:
ah, that makes sense, thanks!